plugins/ingest/src/main/java/org/elasticsearch/ingest/processor/AbstractStringProcessor.java

```diff
@@ -58,10 +56,10 @@ public final void execute(IngestDocument document) {
     public static abstract class Factory<T extends AbstractStringProcessor> implements Processor.Factory<T> {
         @Override
         public T create(Map<String, Object> config) throws Exception {
-            List<String> fields = ConfigurationUtils.readList(config, "fields");
-            return newProcessor(Collections.unmodifiableList(fields));
+            String field = ConfigurationUtils.readStringProperty(config, "field");
```

Comment:
oh right sorry I had missed it's a single value for these processors. sounds good.